kibble/api/translations.go

```diff
@@ -0,0 +1,58 @@
+//    Copyright 2018 SHIFT72
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+	"kibble/models"
+)
+
+// Loads all translations from v1 translations API for the given site if it has the translations_api feature toggle enabled.
+func LoadAllTranslations(cfg *models.Config, site *models.Site) error {
+	if !site.Toggles["translations_api"] {
+		return nil
+	}
+
+	path := fmt.Sprintf("%s/services/users/v1/translations", cfg.SiteURL)
```

Comment:
We've just changed where this api lives... it's going to be `/v1/site_translations`.  Maybe wait until there's been a staging deploy to test it out.